### ISSUE TITLE
Problems with dataset division

### ISSUE THREAD

**Issue by ifei17-hfut** [ASolitaryMan/HFLEA#1]
For `The format of training, validation, and test sources`, is it done according to `This means that the data of four sessions are used as training data; the data of one speaker from the remaining session is used as the validation set and the data of other one speaker forms the testing set.`? Can the authors provide the official format file? I'm stuck at this step when reproducing.

**Comment by ASolitaryMan** [comment]
Thanks for your attention. I update a zip attachment file. I hope it
[IEMOCAP.zip](https://github.com/user-attachments/files/17358983/IEMOCAP.zip)
 can help you.
